.env.example

```
@@ -1,7 +1,7 @@
 MAINNET_RPC=
-MAINNET_DEPLOYER_PK=
+MAINNET_DEPLOYER=
```

wdyt? i think isn't clear enough :/
```suggestion
MAINNET_DEPLOYER_ADDRESS=
```

---

got it wrong, but perhaps we should use a `_ID` or `_NAME` in the .env, or explain what are we doing on the readme 🙏